flamedisx/xenon/x1t_sr1.py

```diff
@@ -38,24 +38,40 @@
 DEFAULT_SINGLE_ELECTRON_GAIN = DEFAULT_G2_TOTAL / DEFAULT_EXTRACTION_EFFICIENCY
 DEFAULT_SINGLE_ELECTRON_WIDTH = 0.25 * DEFAULT_SINGLE_ELECTRON_GAIN
 
-# Official numbers from BBF, do not question them ;-)
+# Official numbers from BBF
 DEFAULT_S1_RECONSTRUCTION_BIAS_PIVOT = 0.5948841302444277
 DEFAULT_S2_RECONSTRUCTION_BIAS_PIVOT = 0.49198507921078005
+DEFAULT_S1_RECONSTRUCTION_EFFICIENCY_PIVOT = -0.31816407029454036 
 
 ##
-# Loading Pax reconstruction bias
+# Combined cuts acceptances
 ##
+path_cut_accept_s1 = ['S1AcceptanceSR1_v7_Median.json']
+path_cut_accept_s2 = ['S2AcceptanceSR1_v7_Median.json']
+
+##
+# Pax reconstruction bias (do not reorder: Lowers, Uppers)
+## 
 path_reconstruction_bias_mean_s1 = ['ReconstructionS1BiasMeanLowers_SR1_v2.json',
-        'ReconstructionS1BiasMeanUppers_SR1_v2.json']
+                                    'ReconstructionS1BiasMeanUppers_SR1_v2.json']
 path_reconstruction_bias_mean_s2 = ['ReconstructionS2BiasMeanLowers_SR1_v2.json',
-        'ReconstructionS2BiasMeanUppers_SR1_v2.json']
+                                    'ReconstructionS2BiasMeanUppers_SR1_v2.json']
+
+##
+# Pax reconstruction efficiencies (do not reorder: Lowers, Medians, Uppers)
+##
+path_reconstruction_efficiencies_s1 = ['RecEfficiencyLowers_SR1_70phd_v1.json',
+                                       'RecEfficiencyMedians_SR1_70phd_v1.json',
+                                       'RecEfficiencyUppers_SR1_70phd_v1.json']
```

Comment:
If the order of the filenames if important I would consider using a dict instead of a list. That way you'll always get the right file for a given key instead of relying on the index in the list. With a dict you can also combine all the different global `path_*` variables for cut acceptances, biases and efficiencies and have them neatly in one place.
It does mean you have to change 'read_maps_tf' to work with the dict.